astropy/constants/constant.py

```diff
@@ -75,10 +75,33 @@ def wrapper(self, *args, **kwargs):
 
 
 class Constant(Quantity, metaclass=ConstantMeta):
-    """A physical or astronomical constant.
+    """
+    A physical or astronomical constant.
```

Comment:
I never understood the endline after the triple quotes. What does it do to the rendered result?
```suggestion
    """A physical or astronomical constant.
```